Makefile

```diff
@@ -19,8 +19,8 @@ ifndef VERSION
 VERSION := $(shell echo `git rev-parse --abbrev-ref HEAD`-`git log -1 --pretty=format:%h`-`date "+%d.%b.%Y.%H.%M.%S"`)
 endif
 
-REGISTRY ?= dpcpinternal
-JUMP_PAD_IMAGE = $(REGISTRY)/jump-pad
+REGISTRY ?= gcr.io/velero-gcp
+JUMP_PAD_IMAGE = $(REGISTRY)/kibishii-jump-pad
```

Comment:
Do we need to modify image name of jump-pad?

Reply:
I'll change it back.